apps/shopifypartners/manifest.yaml

```diff
@@ -0,0 +1,8 @@
+---
+id: shopify-partners
+name: Shopify Partners
```

Comment:
I think we'll need to consider renaming this app before merging. It's not using Shopify's official API, so perhaps it'd be more appropriate to name it "Partnermetrics", or something like that?

Reply:
The app doesn't use Shopify partner API however it is displaying data from shopify parter dashboard. I think this name is appropriate based on the data displayed.

Reply:
Ok, I understand. However, we do need the name changed to merge. It's important for us to minimize any confusion with the official Shopify apps, since we've plenty of new users coming into the ecosystem specifically to use these.